chromadb/test/ef/test_vertex_ef.py

```diff
@@ -0,0 +1,24 @@
+from os import getenv
+
+import pytest
+from chromadb.utils.embedding_functions import GoogleVertexEmbeddingFunction
+
+
+def test_api_key() -> None:
```

Comment:
This will always fail in CI as we don't have the env var. I may have expressed myself poorly about negative test cases.

What I imagined is such:

```python
def test_vertex_ai_missing_api_key() -> None:
     with pytest.raises(match="API Key is invalid"):
        vertex_ef = GoogleVertexEmbeddingFunction(None,
                                               'textembedding-gecko-multilingual',
                                               getenv('PROJECT_ID'))
```

Reply:
OK, I get it. I will revise it.